src/datasources/db/v2/postgresql-logger.service.ts

```diff
@@ -0,0 +1,65 @@
+import { Injectable } from '@nestjs/common';
+import { AbstractLogger, type LogLevel, type LogMessage } from 'typeorm';
+
+@Injectable()
+export class PostgresqlLogger extends AbstractLogger {
+  /**
+   * Write log to specific output.
+   */
+  protected writeLog(
+    level: LogLevel,
+    logMessage: LogMessage | LogMessage[],
+  ): void {
+    const messages = this.prepareLogMessages(logMessage, {
+      highlightSql: false,
+    });
+
+    for (const message of messages) {
+      switch (message.type ?? level) {
+        case 'log':
+        case 'schema-build':
+        case 'migration':
+          console.log(message.message);
```

Comment:
Looking at the types, this can also theoretically have a `prefix`.

Do you think we move these cases to the bottom of the `switch` and also make it the `default` handling as well? I'd like to think no other `type` would be added but just in case.

Reply:
This is the implementation by `TypeOrm`, I prefer not to modify it since we only need to override the `logQuery` method.
https://orkhan.gitbook.io/typeorm/docs/logging

Reply:
Ok, agreed. Can you please add a comment with that link?

Reply:
Of course.